### ISSUE TITLE
Error in getting started documentation?

### ISSUE THREAD

**Issue by codemmmmm** [neo4j-contrib/neomodel#692]
In the [documentation](https://neomodel.readthedocs.io/en/latest/getting_started.html#relationships) `for p in germany.inhabitant.all():` is executed.

The listed example never defines Country.inhabitant though, I suspect this is a mistake and should be added (I think by using cypher?).

**Comment by aanastasiou** [comment]
Indeed, it should be added.

I don't understand what you mean by "cypher". 

The definition of `Country` is [here](https://neomodel.readthedocs.io/en/latest/getting_started.html#defining-node-entities-and-relationships) and it would need a `OneOrMore` relationship `inhabitants` towards entities of type `Person`. Do we agree on this or did you have something else in mind?

**Comment by codemmmmm** [comment]
I agree but I don' know what the proper way is. Because here https://github.com/neo4j-contrib/neomodel/issues/420#issuecomment-469634230 you said that there shouldn't be two relationships defined between `Country` and `Person` (as it is also mentioned in the documentation). That's why I assumed it should use ["cypher"](https://neomodel.readthedocs.io/en/latest/cypher.html). But I am sure you know which is the proper way

**Comment by aanastasiou** [comment]
Alright, let's get some clarity:

The [getting started](https://neomodel.readthedocs.io/en/latest/getting_started.html#defining-node-entities-and-relationships) page, uses an example schema throughout, that is first defined in section "Defining Node Entities and Relationships". So, when the reader hits the section "Relationships", the `inhabitants` property has not been defined anywhere previous and this can indeed cause confusion.

That bit you have flagged correctly and if you can provide a PR that improves it, that would be fantastic.

The discussion in #420 is slightly different. Indeed, it is best to avoid reciprocal relationships that mean the same thing because you are increasing the amount of relationships that the DBMS has to shift through for a given query, beyond what is necessary.

One possible way to fix that part in the documentation without having to modify the examples extensively is to start the "Relationships" section with something like:  "Given a slightly different definition for Person, Country as: [provide the two class definitions with `inhabitants` for Country and Person without a `country`], the following code demonstrates working with relationships using neomodel". Feel free to modify this as you see fit, this is just an example.

Hope this is clearer
